File: assets/code-variants/ch12/23.py
ACCOUNT_1 = Account(state=AccountState.OPEN, balance=50)

ACCOUNT_2 = Account(state=AccountState.CLOSED, balance=0)

ITEM = Item(name="Cheeseburger", price=100)

# Hundreds of lines of other tests...

def test_can_buy_item_returns_false_for_closed_accounts():
    assert not store.can_buy_item(ITEM, ACCOUNT_1)

def test_can_buy_item_returns_false_when_balance_insufficient():
    assert not store.can_buy_item(ITEM, ACCOUNT_2)

File: assets/code-variants/ch12/23.java
private static final Account ACCOUNT_1 = Account.newBuilder()
    .setState(AccountState.OPEN).setBalance(50).build();

private static final Account ACCOUNT_2 = Account.newBuilder()
    .setState(AccountState.CLOSED).setBalance(0).build();

private static final Item ITEM = Item.newBuilder()
    .setName("Cheeseburger").setPrice(100).build();

// Hundreds of lines of other tests...

@Test
public void canBuyItem_returnsFalseForClosedAccounts() {
    assertThat(store.canBuyItem(ITEM, ACCOUNT_1)).isFalse();
}

@Test
public void canBuyItem_returnsFalseWhenBalanceInsufficient() {
    assertThat(store.canBuyItem(ITEM, ACCOUNT_2)).isFalse();
}
